docs/add-ins/addin-tutorial.md

```diff
@@ -962,7 +1068,7 @@ Now that the UI is implemented, let's add the code behind it. Create a file name
         $('#not-configured').show();
       }
 
-      // When insert button is clicked, build the content
+      // When insert button is chosen, build the content
```

Comment:
maybe "selected" instead of "chosen"

Reply:
Good suggestion; I'll make this change throughout the article.